app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,39 @@
+module API
+  module V1
+    class RentsController < ApplicationController
+      include Wor::Paginate
+
+      before_action :authenticate_user!, :find_user
+
+      @user = nil
+
+      def index
+        rents = Rent.where(user: @user)
+        render_paginated rents, each_serializer: Rents::IndexSerializer
+      end
+
+      def create
+        rent = Rent.new(parse_create_rent)
+
+        if rent.save
+          render json: rent, serializer: Rents::CreateSerializer, status: :created
+        else
+          render json: rent.errors
+        end
+      end
+
+      private
+
+      def find_user
+        @user = User.find(params[:user_id])
```

Comment:
You use handle users with `devise_token_auth`, and use the `current_user` method

Reply:
nop, because the user may isn't the same of the session, because the user id is passed by URL parms...well I don't understant why in the URL pass the user_id if I have the session, but that's the reason why I did this method